src/Views/Components/AdminUI/Button/style.module.scss

```diff
@@ -0,0 +1,76 @@
+.button {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    font-size: 1rem;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: 1.5;
+    letter-spacing: normal;
+    border-radius: var(--givewp-rounded-4);
+    cursor: pointer;
+    border: 1px solid transparent;
+
+    > svg {
+        transform: translate(8px, 1px);
+    }
+}
+
+.primary {
+    color: var(--givewp-shades-white);
+    background: var(--givewp-blue-500);
+
+    &:hover {
+        background-color: var(--givewp-blue-400);
+    }
+
+    &:active {
+        background-color: var(--givewp-blue-600);
+    }
+}
+
+.secondary {
+    background: var(--givewp-shades-white);
+    color: var(--givewp-blue-500);
+    border-color: var(--givewp-blue-500);
+
+    &:hover {
+        color: var(--givewp-blue-400);
+        border-color: var(--givewp-blue-400);
+    }
+
+    &:active {
+        color: var(--givewp-blue-600);
+        border-color: var(--givewp-blue-600);
+    }
+}
+
+.danger {
+    background-color: var(--givewp-red-500);
+    color: var(--givewp-shades-white);
+    border-color: var(--givewp-red-500);
+
+    &:hover {
+        background-color: var(--givewp-red-400);
+    }
+
+    &:active {
+        background-color: var(--givewp-red-600);
+    }
+}
```

Comment:
You have many things being repeated here. A reduced version of it could be:
```scss
.primary,
.secondary,
.danger {
    &:hover {
        background-color: var(--givewp-blue-400);
    }

    &:active {
        background-color: var(--givewp-blue-600);
    }
}

.primary {
    color: var(--givewp-shades-white);
    background: var(--givewp-blue-500);
}

.secondary {
    background: var(--givewp-shades-white);
    color: var(--givewp-blue-500);
    border-color: currentColor;
}

.danger {
    background-color: var(--givewp-red-500);
    color: var(--givewp-shades-white);
    border-color: var(--givewp-red-500);
}

Reply:
Hey @pauloiankoski, Each of the these css properties & values are just slightly different from each other. The first two button css values are swapped with each property. Im not sure the proposed solution will have the same outcome